test/common/router/header_formatter_test.cc

```diff
@@ -617,6 +617,10 @@ match: { prefix: "/new_endpoint" }
         key: "x-request-start"
         value: "%START_TIME(%s.%3f)%"
       append: true
+    - header:
+        key: "x-request-start"
```

Comment:
Let's rename the key to something else, e.g. `"x-request-start-multiple"`, and add some similar expectations below.

```
EXPECT_TRUE(headerMap.has("x-request-start-multiple"));
EXPECT_EQ("1522796769.123 2018-04-03T23:06:09.123Z 1522796769");
```